java/ql/lib/semmle/code/java/frameworks/android/Android.qll

```diff
@@ -127,3 +127,10 @@ class CreateFromParcelMethod extends Method {
     this.getEnclosingCallable().getDeclaringType().getAnAncestor() instanceof TypeParcelable
   }
 }
+
+/**
+ * A method annotated with the `android.webkit.JavascriptInterface` annotation.
+ */
+class JavascriptInterfaceMethod extends Method {
```

Comment:
Should this be in `WebView.qll` instead, just like `WebViewAddJavascriptInterfaceMethod` and the like?

Reply:
I think that's a reasonable change. It keeps (afaik) everything related to `android.webkit` to `WebView.qll`